test/integration/podtato-head-application/00-install.yaml

```diff
@@ -147,7 +126,7 @@ spec:
         keptn.sh/app: podtato-head
         keptn.sh/workload: podtato-head-left-leg
         keptn.sh/version: 0.1.0
-        keptn.sh/pre-deployment-tasks: check-entry-service
```

Comment:
please do not remove the pre-deployment-task annotations from the deployment. Removing these may be the reason why the integration tests are failing